pennylane/operation.py

```diff
@@ -364,6 +365,20 @@ def __init__(self, *params, wires=None, do_queue=True):
         if wires is None:
             raise ValueError("Must specify the wires that {} acts on".format(self.name))
 
+        # turn wires into Wires object
+        self._wires = Wires(wires)  #: Wires: wires on which the operator acts
+
+        # check that the number of wires given corresponds to required number
```

Comment:
This check was previously in a much longer ``_check_wires`` function that is now unnecessary.